login.php

```diff
@@ -4,157 +4,76 @@
-          <img src="profpraisal.png" alt="Your Image Alt Text">
-        </div>
-        <form action="" id="login-form">
-          <!-- <div class="card">
-    <div class="card-body login-card-body">
-      <form action="" id="login-form"> -->
-          <div class="input-group mb-3">
-            <input type="email" class="form-control" name="email" required placeholder="Email">
-            <div class="input-group-append">
-              <div class="input-group-text">
-                <span class="fas fa-envelope"></span>
-              </div>
-            </div>
-          </div>
-          <div class="input-group mb-3">
-            <input type="password" class="form-control" name="password" required placeholder="Password">
-            <div class="input-group-append">
-              <div class="input-group-text">
-                <span class="fas fa-lock" onclick="togglePasswordVisibility()"></span>
-                <span class="fas fa-eye" style="display:none;" onclick="togglePasswordVisibility()"></span>
-              </div>
+<body>
+    <h1 id="page-title" class="text-center">Login Page</h1>
+    <hr id="title_hr" class="mx-auto">
+    <div id="login-wrapper">
+        <div class="text-muted"><small><em>Please fill in all the required fields</em></small></div>
+        <?php if (isset($error) && !empty($error)): ?>
+            <div class="message-error"><?= $error ?></div>
+        <?php endif; ?>
+        <?php if (isset($_SESSION['msg']['success']) && !empty($_SESSION['msg']['success'])): ?>
+        <div class="message-success">
+            <?php 
+            echo $_SESSION['msg']['success'];
+            unset($_SESSION['msg']);
+            ?>
+        </div>  
+        <?php endif; ?>
+        <form action="" method="POST">
+            <div class="input-field">
+                <label for="email" class="input-label">Email</label>
+                <input type="email" id="email" name="email" value="<?= htmlspecialchars($_POST['email'] ?? "", ENT_QUOTES) ?>" required="required">
             </div>
-          </div>
-          <div class="form-group mb-3">
-            <label for="">Login As</label>
-            <select name="login" id="" class="custom-select custom-select-sm">
-              <option value="3">Student</option>
-              <option value="2">Faculty</option>
-              <option value="1">Admin</option>
-            </select>
-          </div>
-          <div class="row">
-            <div class="col-8">
-              <div class="icheck-primary">
-                <input type="checkbox" id="remember">
-                <label for="remember">
-                  Remember Me
-                </label>
-              </div>
+            <div class="input-field">
+                <label for="password" class="input-label">Password</label>
+                <input type="password" id="password" name="password" value="<?= htmlspecialchars($_POST['password'] ?? "", ENT_QUOTES) ?>" required="required">
```

Comment:
There was some code written by a prev contributor for toggle of password "fas fa-eye" - please don't scrape whatever was written before. I can see the different icons that were there are removed. It is okay to make additions to the file, this looks like is rebranded.